ChummerHub.Client/Model/CharacterExtended.cs

```diff
@@ -159,7 +159,7 @@ internal IList<Tag> PopulateTags()
                 MySINnerFile.SiNnerMetaData.Tags.Remove(f);
             }
             MySINnerFile.SiNnerMetaData.Tags.Add(tag);
-            foreach(var childtag in MySINnerFile.SiNnerMetaData.Tags)
+            foreach(var childtag in MySINnerFile.SiNnerMetaData.Tags.Where(a => a != null).ToList())
```

Comment:
Just curious, why is the `ToList()` necessary? Can `SetSinnerIdRecursive` add or remove elements from `MySINnerFile.SiNnerMetaData.Tags`?